src/components/TermsAndConditions.js

```diff
@@ -16,12 +17,16 @@ class TermsAndConditionsComponent extends Component {
   }
 
   componentDidMount() {
-    if (this.state.devMode) {
-      console.log("[*] dev Environment: no need to check them checkb0xes...");
-      const checkboxDOM = document.getElementById("termsCheckbox");
-      checkboxDOM.checked = true;
-      this.setState({ boxChecked: true });
-      this.clickButton();
+    if (this.props.clientConfiguration.showTermsAndConditions !== true) {
+      this.props.termsSigned();
+    } else if (
+      this.props.clientConfiguration.termsAndConditionsSetAgreedCookie ===
+        true &&
+      readCookie("termsAndConditionsSigned") === "true"
+    ) {
+      this.props.termsSigned();
+    } else if (this.state.devMode) {
+      this.props.termsSigned();
```

Comment:
Skip agreeterm and conditions screen if:
- showTermsAndConditions field in client_configuration is not there or false
-  termsAndConditionsSetAgreedCookie field in client configuration is true and a cookie has been found
- the app is in devmode